packages/web/app/utils/survey.jsx

```diff
@@ -296,16 +298,24 @@ export function getFormInitialValues(
   return initialValues;
 }
 
-export const getAnswersFromData = (data, survey) =>
-  Object.entries(data).reduce((acc, [key, val]) => {
-    if (
-      survey.components.find(({ dataElement }) => dataElement.id === key)?.dataElement?.type !==
-      'PatientIssue'
-    ) {
+export const getAnswersFromData = async (data, survey) =>
+  Object.entries(data).reduce(async (accPromise, [key, val]) => {
```

Comment:
This is a little unusual to me, but it could be a personal preference. I would do something like `.map`, and then an `const entries = await Promise.all(`
```suggestion
  const answerEntryPromises = Object.entries(data).map(async ([key, val]) => {
  ...
  };
  const answerEntries = await Promise.all(entryPromises);
  const answers = Object.fromEntries(answerEntries);
```
Up to you though!

Reply:
Yeah no I just went for the path of least resistance on the rewrite, this pattern is only found on 1 other place in the codebase so I considered it okay. I'll change it up

Reply:
Yeah great call!